Docs/developer_guide/script_repository.rst

```diff
@@ -0,0 +1,4237 @@
+   dicomBrowser.importDirectory(dicomFilesDirectory, dicomBrowser.ImportDirectoryAddLink)
+   # wait for import to finish before proceeding (optional, if removed then import runs in the background)
+   dicomBrowser.waitForImportFinished()
+
+How to import DICOM files using DICOMweb
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Download and import DICOM data set using DICOMweb from `Kheops <https://kheops.online/>`__, Google Health API, etc.
+
+How to obtain accessToken:
+
+-  Google Cloud: Execute ``gcloud auth print-access-token`` once you have logged in
+-  Kheops: create an album, create a sharing link (somethin like ```https://demo.kheops.online/view/TfYXwbKAW7JYbAgZ7MyISf`` <https://demo.kheops.online/view/TfYXwbKAW7JYbAgZ7MyISf>`__), the token is the string after the last slash
+
+.. code-block:: python
+
+   slicer.util.selectModule("DICOM")  # ensure DICOM database is initialized and
+   slicer.app.processEvents()
+   from DICOMLib import DICOMUtils
+   DICOMUtils.importFromDICOMWeb(
+       dicomWebEndpoint="http://demo.kheops.online/api",
+       studyInstanceUID="1.3.6.1.4.1.14519.5.2.1.8421.4009.985792766370191766692237040819",
+       accessToken="TfYXwbKAW7JYbAgZ7MyISf")
+
+How to access top level tags of DICOM images imported into Slicer?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For example, to print the first patient's first study's first series' "0020,0032" field:
+
+.. code-block:: python
+
+   db=slicer.dicomDatabase
+   patientList=db.patients()
+   studyList=db.studiesForPatient(patientList[0])
+   seriesList=db.seriesForStudy(studyList[0])
+   fileList=db.filesForSeries(seriesList[0])
+   # Note, fileValue accesses the database of cached top level tags
+   # (nested tags are not included)
+   print(db.fileValue(fileList[0],'0020,0032'))
+   # Get tag group,number from dicom dictionary
+   import pydicom as dicom
+   tagName = "StudyDate"
+   tagStr = str(dicom.tag.Tag(tagName))[1:-1].replace(' ','')
+   print(db.fileValue(fileList[0],tagStr))
+
+How to access DICOM tags nested in a sequence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+   db=slicer.dicomDatabase
+   patientList=db.patients()
+   studyList=db.studiesForPatient(patientList[0])
+   seriesList=db.seriesForStudy(studyList[0])
+   fileList=db.filesForSeries(seriesList[0])
+   # use pydicom to access the full header, which requires
+   # re-reading the dataset instead of using the database cache
+   import pydicom
+   pydicom.dcmread(fileList[0])
+   ds.CTExposureSequence[0].ExposureModulationType
+
+How to access tag of a volume loaded from DICOM? For example, get the patient position stored in a volume:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+volumeName='2: ENT IMRT'
+   n=slicer.util.getNode(volumeName)
+   instUids=n.GetAttribute('DICOM.instanceUIDs').split()
```

Comment:
```suggestion
    instUids = n.GetAttribute("DICOM.instanceUIDs").split()
```

Reply:
In addition of adding spaces around the `=`, I also suggest we also consistently use double quotes